### PR TITLE
feat: Add Greek localization (M2-7799)

### PR DESCRIPTION
**Pull request by ChaconC**
### 📝 Description

Adds Greek language localization support.

🔗 [Jira Ticket M2-7799](https://mindlogger.atlassian.net/browse/M2-7799)

### 📸 Screenshots

| 1                      | 2                                 |
| -------------------------------------- | ------------------------------------- |
| CleanShot 2024-10-15 at 09 52 35 | CleanShot 2024-10-15 at 09 52 10 |

### 🪤 Peer Testing

- Load Web App
- Select "Ελληνικά" from language dropdown menu

### Notes
A few strings were provided with "deprecated" instructions and are not translated. They will be removed in a future ticket (M2-7879).
Additionally, the translation vendor will be performing visual validation for long or strange strings.

### CODE REVIEW

File: src/i18n/el/translation.json [hunk 1]
```diff
@@ -0,0 +1,333 @@
...
+      "midnight": "Μεσάνυχτα"
+    },
+    "wrondLinkParametrError": "Φτάσατε σε αυτήν τη διεύθυνση URL κατά λάθος. Επικοινωνήστε με τον υπεύθυνο αυτής της βοηθητικής εφαρμογής για περαιτέρω βοήθεια.",
+    "activity_due_date": {
+      "available_all_day": "Διαθέσιμο: Όλη την ημέρα",
+      "scheduled_at": "Scheduled At",
+      "available": "Διαθέσιμο από",
+      "to": "έως",
+      "day": "day",
+      "days": "days",
```
Comment (LashaunnaS) [inline src/i18n/el/translation.json:54]: Are these the same in Greek?

File: src/i18n/el/translation.json [hunk 2]
```diff
@@ -0,0 +1,333 @@
...
+      "networkError": "Σφάλμα δικτύου. Επιστροφή ",
+      "invitationAlreadyRemoved": "Η πρόσκλησή σας έχει ήδη αφαιρεθεί.",
+      "invitationAlreadyAccepted": "Η πρόσκληση έχει γίνει ήδη αποδεκτή",
+      "notAssociatedAccount": "Η πρόσκληση αυτή δεν σχετίζεται με τον λογαριασμό στον οποίον είστε συνδεδεμένοι αυτήν τη στιγμή. Συνδεθείτε στον λογαριασμό που σχετίζεται με το email όπου λάβατε την πρόσκληση.",
+      "invitationDeclined": "Η πρόσκληση έχει απορριφθεί",
+      "home": "Αρχική σελίδα",
+      "invitationAccepted": "Η πρόσκληση έγινε αποδεκτή",
+      "viewInvitation": "to View Invitation",
+      "acceptInvitation": "to Accept this Invitation!",
+      "declineInvitation": "to Decline This Invitation",
```
Comment (LashaunnaS) [inline src/i18n/el/translation.json:119]: and these?

File: src/i18n/el/translation.json [hunk 3]
```diff
@@ -0,0 +1,333 @@
...
+        "step2_1": " (εάν έχετε λογαριασμό MindLogger) ή εγγραφείτε (αν είστε νέοι στο MindLogger). Για να εγγραφείτε, πατήστε «Νέος χρήστης» στην οθόνη σύνδεσης και εισαγάγετε τη διεύθυνση email όπου λάβατε αυτήν την πρόσκληση.",
+        "step3": "Πατήστε <strong>{{displayName}}</strong> στην αρχική οθόνη του MindLogger και είστε έτοιμοι! Εάν δεν εμφανίζεται το <strong>{{displayName}}</strong>, ανανεώστε την οθόνη σύροντας το δάχτυλό σας από πάνω προς τα κάτω και θα εμφανιστεί ένας περιστρεφόμενος τροχός κατά τη φόρτωση του <strong>{{displayName}}</strong>.",
+        "footer": "Το Child Mind Institute αποτελεί τον δημιουργό του MindLogger, μιας πλατφόρμας για τη δημιουργία βοηθητικών εφαρμογών, ωστόσο δεν είναι υπεύθυνο για το περιεχόμενο των βοηθητικών εφαρμογών που δημιουργείται από εξωτερικά μέρη.",
+        "loading": "Φόρτωση συνδέσμου πρόσκλησης",
+        "reviewers": "Users who can see your data for this applet:",
+        "managers": "Users who can change this applet's settings, including who can access your data:",
+        "coordinators": "Users who can change this applet's settings, but who cannot change who can see your data:",
```
Comment (LashaunnaS) [inline src/i18n/el/translation.json:138]: and these?

**Review by LashaunnaS — COMMENTED** [review]
Noticed a few untranslated lines.

File: src/i18n/el/translation.json (hunk 3 above)
Comment (ChaconC) [reply src/i18n/el/translation.json:138]: This was provided with a "deprecated/not used" instruction, will be cleaned up in the near future by [M2-7879](https://mindlogger.atlassian.net/browse/M2-7978)

File: src/i18n/el/translation.json (hunk 2 above)
Comment (ChaconC) [reply src/i18n/el/translation.json:119]: This was provided with a "deprecated/not used" instruction, will be cleaned up in the near future by [M2-7879](https://mindlogger.atlassian.net/browse/M2-7978)

File: src/i18n/el/translation.json (hunk 1 above)
Comment (ChaconC) [reply src/i18n/el/translation.json:54]: Thanks for taking a look at these translations @LashaunnaS! The translation file was provided with instructions on a few items which didn't require translation due to being legacy or unused strings. I'll update the PR description for clarity.

**Review by LashaunnaS — APPROVED** [review]

**Pull request merged**